tests/test_browsing_context.py

```diff
@@ -786,3 +786,19 @@ async def test_browsingContext_waitForSelectorWithMissingElement_failedWithTimeo
 @pytest.mark.skip(reason="TODO: Not implemented yet.")
 async def test_browsingContext_clickElement_clickProcessed():
     pass
+
+
+@pytest.mark.asyncio
+async def test_browsingContext_reload(websocket, context_id, html):
+    url = html()
+
+    await goto_url(websocket, context_id, url)
+
+    result = await execute_command(websocket, {
```

Comment:
We should verify that the actual reload eventually happened. You can use `test_browsingContext_navigateWait...`  as blueprints.

Reply:
Done